metadata-ingestion/src/datahub/ingestion/source/usage/bigquery_usage.py

```diff
@@ -392,6 +421,18 @@ class BigQueryUsageConfig(DatasetSourceConfigBase, BaseUsageConfig):
     log_page_size: Optional[pydantic.PositiveInt] = 1000
     query_log_delay: Optional[pydantic.PositiveInt] = None
     max_query_duration: timedelta = timedelta(minutes=15)
+    credential: Optional[BigQueryCredential]
+    credentials_path: Optional[str] = None
```

Comment:
nit: Optional implies None is the default value. So, can be removed like the line above.